test/test_ci_container.rb

```diff
@@ -29,13 +29,12 @@ def setup
       }
     end
 
-    # Chdir to root, as Containment will set the working dir to PWD and this
-    # is slightly unwanted for tmpdir tests.
-    Dir.chdir('/')
-
     @job_name = self.class.to_s
     @image = 'ubuntu:15.04'
-    VCR.turned_off { cleanup_container }
+    VCR.turned_off do
+      cleanup_container
+      Docker::Image.create(fromImage: @image)
```

Comment:
Image creation should not be done when in playback mode. (see vcr_it in container upgrade test or smth)